src/features/notifications/NoNotifications.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import Text from '../../components/Text'
+import BellBubble from '../../assets/images/bellBubble.svg'
+import Box from '../../components/Box'
+
+const NoNotifications = () => {
```

Comment:
could maybe call this `EmptyNotifications` or something? The `NoNo` in `NoNotifications` kind of runs together when I skim it 😅

Reply:
ok, I'll rename it in my next PR, if that's all right.

Reply:
np